games-simulation/simutrans/simutrans-nightly.recipe

```diff
@@ -9,18 +9,17 @@ changes which make your savegames unreadable by any official version, or \
 bugs that will destroy your game, or not even run. They are for bug testing \
 only."
 HOMEPAGE="http://www.simutrans.com"
-SOURCE_URI="svn://tron.yamagi.org/simutrans/simutrans/trunk \
-	--username=anon --password="
+SOURCE_URI="svn://servers.simutrans.org/simutrans/trunk"
 SOURCE_FILENAME="simutrans-$portVersion"
 SOURCE_DIR="trunk"
 COPYRIGHT="1997-2004 Hj. Malthaner
-	2005-2015 The Simutrans Team"
+	2005-2017 The Simutrans Team"
 LICENSE="Artistic"
-REVISION="2"
+REVISION="3"
 ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 PROVIDES="
-	simutrans$secondaryArchSuffix = $portVersion
+	simutrans$secondaryArchSuffix = $portVersion compat >= 120.2
```

Comment:
Thank you for the link. However, I still have some questions:
1.) "If no compatibility version is specified, the resolvable can only satisfy dependency constraints with an exactly matching version."
Does that mean that if I have a package with version 1.0.0 (and there is no compatibility declared in PROVIDES for "compat >= 1.0") it can't fullfill a dependency that is declared in REQUIRES as ">= 1.0"?
2) Okay, 1.2.3 > 1.2.3~nightly, but 1.2.3~nightly > 1.2.2 which means that haikuporter would still build the nightly instead of the stable 1.2.2 if you don't specify the version to build?